File: VB/Customer.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic

Namespace WindowsApplication1
	Public Class Customer
		Public Sub New()

		End Sub

		' Fields...
		Private _ID As Integer
		Private _B As String
		Private _LastName As String
		Private _FirstName As String


		Public Property FirstName() As String
			Get
				Return _FirstName
			End Get
			Set(ByVal value As String)
				_FirstName = value
			End Set
		End Property


		Public Property LastName() As String
			Get
				Return _LastName
			End Get
			Set(ByVal value As String)
				_LastName = value
			End Set
		End Property




		Public Property ID() As Integer
			Get
				Return _ID
			End Get
			Set(ByVal value As Integer)
				_ID = value
			End Set
		End Property


	End Class
End Namespace

File: CS/Customer.cs
using System;
using System.Collections.Generic;

namespace WindowsApplication1
{
    public class Customer
    {
        public Customer()
        {

        }

        // Fields...
        private int _ID;
        private string _B;
        private string _LastName;
        private string _FirstName;


        public string FirstName
        {
            get { return _FirstName; }
            set
            {
                _FirstName = value;
            }
        }


        public string LastName
        {
            get { return _LastName; }
            set
            {
                _LastName = value;
            }
        }




        public int ID
        {
            get { return _ID; }
            set
            {
                _ID = value;
            }
        }
        
        
    }
}
